### DISCUSSION TITLE
how to handle z-index on mouse over with custom maker and marker cluster

### DISCUSSION THREAD

**Discussion by select-good-way** [inocan-group/vue3-google-map#211] [Q&A]
how to handle z-index on mouse over with custom maker and marker cluster 
1) when cursor over the maker which in below the other marker then show the marker above the other markers by increasing z-index

2) when mouse cursor over the the cluster then show marker above the other cluster or marker

please assist me how to achieve this, I was trying to increase z-index by apply mouse over event its working if two clusters are overlapped each other but for custom markers it's not working if cluster icon is below any custom marker then it's not coming forward to the custom marker

**Comment by HusamElbashir** [comment]
There is no easy way to do this unfortunately. We need to place custom markers on the [overlayMouseTarget](https://developers.google.com/maps/documentation/javascript/reference/overlay-view#MapPanes.overlayMouseTarget) pane to receive DOM events but regular markers exist on the [markerLayer](https://developers.google.com/maps/documentation/javascript/reference/overlay-view#MapPanes.markerLayer) pane which is below.

**Comment by select-good-way** [comment]
instead on custom marker now I am using marker mouse over and out not working for increasing z index
